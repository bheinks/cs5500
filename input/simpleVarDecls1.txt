(*
	Single declarations of simple types
*)

program p;

var x : integer;
    y : boolean;
    z : char;

begin
  write(5)
end.

