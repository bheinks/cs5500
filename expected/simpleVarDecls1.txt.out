

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding x to symbol table with type INTEGER

+++ Adding y to symbol table with type BOOLEAN

+++ Adding z to symbol table with type CHAR

<<< Exiting scope...

---- Completed parsing ----
